### DISCUSSION TITLE
Neu im Projekt / Welcher Hoymiles?

### DISCUSSION THREAD

**Discussion by chiquadrat14** [hoylabs/OpenDTU-OnBattery#711] [General]
Hallo zusammen,

ich bin vor kurzem auf dieses coole Projekt gestoßen, da ich gerade plane, mein bisheriges "Balkonkraftwerk" auf ein "Balkonkraftwerk XXL" mit Speicher auszubauen und natürlich auch eine Nulleinspeisung erreichen möchte. Im Netz bin ich auch noch auf eine alternative Lösung mit einem Sun 1000 WR/Lumentree gestoßen (https://github.com/trucki-eu/Trucki2Shelly-Gateway) aber irgendwie habe ich das Gefühl, dass dort das gesamte Projekt an einer Person hängt und auch in letzter nicht mehr soviel passiert...in diesem Projekt scheint richtig viel "Leben" zu sein...

Ich stelle mir gerade die nötigen Komponenten zusammen (danke auch für die Praxisbeispiele im Wiki!). Auch wegen der Beispiele habe ich mich für den Pylontech Akku entschieden.

An einem Punkt bin ich mir noch unsicher: Da ich mir einen neuen Wechselrichter von Hoymiles kaufen muss, stellt sich die Frage, welches Modell ich wählen sollte, damit es möglichst optimal mit OpenDTU on Battery funktioniert. Eine klare Tendenz für oder gegen ein bestimmtes Hoymiles Modell konnte ich aus den Diskussionen bisher nicht herauslesen. In den Praxisbeispielen wird häufig der HM-1500 angeführt. Bei den HMS Modellen braucht man wohl ein anderes Antennenmodul im ESP32, aber ich habe verstanden, dass es dann prinzipiell genauso gut funktioniert...

Ich habe an anderer Stelle gelesen, dass der HM 1500 zwar sehr gut ist, aber eher Probleme mit Übertemperatur hat als z.B. der (neuere) HMS 1600.

Daher tendiere ich aktuell dazu, den HMS 1600 zu wählen.

Falls Ihr noch weitere Gedanken zur Wahl des WR habt, freue ich mich über euer Feedback!

Dann hoffentlich bis Bald mit einem funktionierenden OpenDTU on Battery System!

Grüße

Philipp

**Comment by rs3hc** [comment]
Willkommen.

Wenn du nicht viel "basteln" willst lege ich dir die Platine von @alexseuf ans Herz --> [https://www.akkudoktor.net/forum/migrated-forums-balkonsolar/opendtu-on-battery-platine-mit-maximal-ausbau-dc-dc-und-jk-bms-stecker/paged/2/#post-172350](https://www.akkudoktor.net/forum/migrated-forums-balkonsolar/opendtu-on-battery-platine-mit-maximal-ausbau-dc-dc-und-jk-bms-stecker/paged/2/#post-172350)

**Reply by schlimmchen** [reply]
Oder: https://shop.allianceapps.io/products/opendtu-fusion-community-edition

**Reply by JoSoBay** [reply]
Kann ich den Pylontech Speicher und den Victron MPPT direkt an das OpenDTU Fusion-Board anschliessen? Ich meine CANH from Pylontech und CANL from Pylontech bzw. GND, 5V, TX vom Victron direkt anschließbar an das Fusion-Board?

**Reply by schlimmchen** [reply]
Einen Victron MPPT über Ve.Direct solltest du über einen ADUM1201 anschließen, es würde aber auch direkt gehen (GND, RX, TX). Letzteres ist aber Murks und verlässt sich darauf, dass die ESP32 Pins 5V tolerant sind, was sie nur inoffiziell sind.

Um mit einer Pylontech über CAN zu sprechen brauchst du einen CAN Controller. Schau mal im Wiki.

**Reply by JoSoBay** [reply]
Danke. Hab mir die passenden Elemente bestellt. 👍

**Comment by swingstate** [comment]
HM-1500 seit einem Jahr im Einsatz hier mit 4x 385W + Batterie. Selbst im Hochsommer keine Probleme. Der WR ist im Schatten angebracht.

**Reply by Artikel14** [reply]
Hallo chiquadrat14 und Kollegen,
einen hochwertigen LFP-Akku (zum Beispiel mit über 2550 Wattstunden) habe ich selbst so erweitert, getestet und erfolgreich im Betrieb, dass alle "überschüssig" erzeugte Energie erst mal extrem effizient gespeichert wird. Man kann über diese Konstruktion wahlweise Tag und Nacht die Grundlast ins Hausnetz einleiten oder sogar lastabhängig hervorragend die Einspeisung steuern.

Vorteil Nummer 1: Man kann viel mehr von dem erzeugten Solarstrom Tag und Nacht nutzen.
Vorteil Nummer 2: Man kann viel mehr Solarmodule sinnvoll anschließen (Erzeugungsleistung).
Vorteil Nummer 3: Man kann es weiterhin ohne Handwerker selbst anschließen und zulässig als "Balkonkraftwerk" betreiben.
Vorteil Nummer 4: Man kann die Einspeisung nach dem Verbrauch einstellen und Tag und Nacht unterschiedlich steuern.
Vorteil Nummer 5: Man muss nicht die Verbraucher bei Sonne einschalten und abschalten, wenn eine Wolke kommt. Mitbewohner oder Familie müssen sich nicht umstellen.
Vorteil Nummer 6: Man kann die gespeicherte Energie auch an einem nachfolgenden Regentag weiternutzen.
Vorteil Nummer 7: Man kann viel mehr von dem selbsterzeugten Strom selbst nutzen.
Vorteil Nummer 8: Auch für die Umwelt entsteht in der Praxis ein größerer Nutzen (viel weniger Leitungsverluste und mehr CO2-Einsparung).
Vorteil Nummer 9: Alle derzeitigen Vorschriften können eingehalten werden.
Vorteil Nummer 10: Ist günstig erweiterbar als Notstromquelle bei Netzausfall (Solargenerator).

Sehr verschiedene Konfigurationen habe ich als Ingenieur aufwändig getestet. Meine Konstruktion ist deutlich flexibler und effizienter als bisherige "kommerzielle" Angebote (auch vom Preis). Bestehende Balkonkraftwerke kann man durchaus mit einem hocheffizienten Akku nachrüsten und bei Bedarf erweitern.

(1) Solarmodule bis zu 1560 Watt mit einem Laderegler bis 3000 Watt mit zwei Ladereglern
(2) Solarkabel zum MPPT Laderegler
(3) MPPT Laderegler (mit Leistungsanzeige)
(4) Spezialkabel zur Steuerungsbox (Angepasste/erweiterte Version von openDTU-onBattery)
(5) Spezialkabel zum Akku
(6) Akku 2550+ Wh mit Bluetooth (Handy und Steuerungsoption)
[optional währen auch 5800Wh möglich oder jeweils ein Vielfaches bis vier Stück parallel]
[genutzt werden 80% der Bruttokapazität um über 5000 Ladezyklen nutzen zu können]
(7) Solarkabel von der Steuerungsbox zum Wechselrichter Hoymiles z.B. HM-600
(wahlweise auch alle mit openDTU kompatiblen Wechselrichter)
(8) Anschlussleitung Kabel Betteri Stecker BC01 zur Schuko-Steckdose
(9) Zum Beispiel kombinierbar mit Fritz 210 zur Ertragsmessung auf der Einspeiseseite

Die Leistung wird dabei fünfmal gemessen:
(A) Solarleistung am MPPT Laderegler (DC)
(B) Lade-/Entladeleistung im Akku (DC)
(C) Eingangsleistung Hoymiles (DC)
(D) Einspeiseleistung Hoymiles (AC)
(E) Einspeiseleistung und Statistik Fritz 210 (AC)

Leistungen (B bis E) sind auf dem iPhone oder Android anzeigbar.

Optimieren macht mir Spaß, auf den bestmöglichen hohen Wirkungsgrad. Wirtschaftlich habe ich zusätzlich bei uns regional noch eine Förderung für meinen Solarspeicher beantragt [und auch genehmigt bekommen 50 Prozent] sowie eine Förderung für mein „Balkonkraftwerk“. Aber auch völlig ohne Förderung rechnet es sich wie zwei bis drei normale Balkonkraftwerke, weil man Stromüberschüsse kaum noch verschenkt, vom besseren Umweltschutz durch entfallende Leitungsverluste ganz zu schweigen.

Laderegler nutze ich einen PowMr MPPT Controller (bis zu 98%Wirkungsgrad).
Getestet habe ich zusätzlich auch Victron Smartsolar Laderegler mit bis zu 99% Wirkungsgrad. Durch den hohen Wirkungsgrad habe ich den völlig geräuschlosen Betrieb ohne Lüfter (nur mit passiven Kühlkörpern) in beiden Fällen erreicht.

Den selbst erzeugten Solarstrom nutzt man am wirkungsvollsten,
wenn man den erzeugten Stromüberschuss mit einem gut
optimierten Zwischenspeicher für den Nachtverbrauch sowie Lücken durch Wolken und Regentage speichert.

Die Zellen Speichers halten gemäß Herstellerangaben über 5000 Zyklen.
Bei 2400 kWh Stromverbrauch im Jahr (Beispiel) und vier Solarpanelen rechnet sich der mittelgroße 2,55 Akku nach weniger als 6 Jahren und das Gesamtsystem nach rund 7 Jahren (siehe Screenshot).
Der sehr große 5,850 Akku rechnet sich bei mir nach rund 9 Jahren und das Gesamtsystem auch nach rund 9 Jahren.

Gerechnet ist das mit nur 0,29 Euro pro Kilowattstunde, obwohl in der Vergangenheit der Strompreis meist gestiegen ist.

Vorteil: Der 2,55+ kWh Akku rechnet sich also deutlich schneller, obwohl er etwas mehr je kWh kostet.

Aber der größere Akku (5,85+ kWh) ermöglicht noch mehr eigengenutzten Solarstrom und rechnet sich nach der Amortisationszeit von 9 Jahren besser, weil man damit auf Dauer weiterhin mehr Strom einspart und somit auf Dauer mehr Gewinn macht. Außerdem wird der größere Akku weniger ausgenutzt und dürfte daher gemäß den Zyklus Daten für LFP-Akkus deutlich über 5000 Zyklen halten. Würde mich nicht wundern, wenn der nach über 20 Jahren (oder sogar mehr) noch hervorragend funktioniert und Gewinne abwirft.

Je mehr die Strompreise über 29 Cent steigen, umso schneller und umso mehr lohnt es sich.

Die Verluste habe ich in vielen Tests von Konfigurationen auf wenige Prozent reduziert:
Optimiert sind die Akkuzellen mit im Niederlastbereich mit circa nur 1 Prozent jeweils beim Einspeichern in den Akku und beim Entnehmen aus dem Akku.
Optimiert auch auf nur ca. 3 bis 2 Prozent beim Laderegler.
Optimiert auch die Hoymiles Wechselrichter, diese haben nach aktuellen Messungen den höchsten Wirkungsgrad von ca. 95 (bis zu 97) Prozent. Sogar die Kabel habe ich nach Preis/Leistung optimiert.

Bereit wäre ich, einen meiner gebauten Prototypen abzugeben, wenn nach 1 Monat und nach 12 Monaten ein kurzer Bericht geschrieben wird, über die Erfahrungen.

Es ist ein mehrdimensional optimierter Solarspeicher (optimiert auf Preis, Wirkungsgrade, Leistung und Haltbarkeit). Das nenne ich jetzt mal "OptiExa".
Gerade in der Übergangszeit scheint die Sonne mit Lücken, teilweise mehrfach kurze Zeiten am Tag. Mit den vier Modulen (oder sogar mehr) kann man mehr Energie auffangen und dann Tag und Nacht die Grundlast oder auch einstellbar flexibel einspeisen. Mehr als 600 bis 800 Watt sind AC-seitig gar nicht nötig. Ein BasisSet könnte man schon für 950 Euro anschlussfertig für Hoymiles-600 benutzen mit über 2550 Wattstunden Brutto-Kapazität.

Man kann an den Speicher fast alle großen Module anschließen,
(fast alle üblichen "Balkonsolar"-Module),
die typischerweise ab 350 bis 490 Watt peak und >30 Volt "Modulspannung" haben.

Auch die Module (typischerweise zwei bis acht Stück) kann man gemäß den technischen und sonstigen Daten optimieren nach Wirkungsgrad, Preis, lange Haltbarbeit, Leistung Wattp, Schwachlichtfähigkeit, Optik/Farbe, Teilverschattungsfähigkeit, Glasmodule/bifacial/shingled, Geometrie (Länge/Breite), Logistik/Entfernung/Lieferbarkeit, Garantiezusagen, Kunden-Rezessionen sowie Vertrauen in Lieferant und Hersteller.

Ich selbst hatte Jinkosolar und Bluesun Module "Black" und "fullblack" in die engere Auswahl gezogen, weil über den höheren Wirkungsgrad und weitere Qualitäten wie die längere Haltbarkeit sich der geringe Mehrpreis für mich mehr als rechnet. Aber auch mit sehr preiswerten Modulen kann man sein persönliches Optimum finden. Bei uns sind jetzt Bluesun shingled "fullblack" (auch wegen der Geometrie) mit einem OptiExa-Speicher-Set im Betrieb und ich bin jeden Tag bisher zufrieden.

Andere Konzepte haben neben der etwas geringeren Kapazität nach meinen Erfahrungen und Messungen einen niedrigeren Wirkungsgrad (deutlich mehr Energieverluste durch die notwendigen Spannungswandlungen). Bei weniger Optimierung und somit schlechterem Wirkungsgrad sind meist ein oder mehrere Lüfter nötig, also kein geräuschloser Betrieb. 
Die Ladeströme und Entladeströme sind oft stärker beschränkt und stressen daher die Akkuzellen mehr. Insbesondere nachträgliche Erweiterungen sind deutlich kostenaufwändiger.
Geringere Kapazität bedeutet deutlich weniger Puffer, und damit mehr Belastungen im Grenzbereich der Kapazität zu lasten der Haltbarkeit (Zyklenfestigkeit). Oft geringere Ladeleistung (800 bis 1000W gegenüber 1560W/3000W).

In meinem Speicher-Konzept (OptiExa-LFP-Set) ist neben dem Speicher mit BMS & Balancer ein guter Laderegler sowie die DTU zur Steuerung für Grundlast, Lasterhöhung und Last nach Bedarf und die Basiskabel bereits beinhaltet. 1560W Laderegler (bis vier typ Module) ist im Set dabei, sehr leicht erweiterbar für rund 100 Euro sogar auf 3000 Watt peak Ladeleistung durch zweiten Laderegler am gleichen Speicher. Dann sind bis zu acht typ. Solarmodule anschließbar.
Zusätzlicher Akku kann nachgerüstet werden bis zu 10.24kWh (jeweils bis vier Stück)
Nach vielen Konzeptionen, Prototypen, Tests und Optimierungen bin ich jetzt hochzufrieden. Bis 97/98 % Wirkungsgrad des Ladereglers. Bis 99% Wirkungsgrad beim Einspeichern in den Akku und beim Ausspeichern (100% durch automatischen Passthrough tagsüber). 97 bis (meist) 95,5% Wirkungsgrad der Hoymiles Wechselrichter. Dies ist aber sowieso normal auch ohne Speicher. Alles bleibt schön kühl und völlig leise (denn auch im Grenzbereich völlig lüfterlose Kühlung auch über die optimierten (bewährten) großen Kühlkörper des Ladereglers). Wifi-Steuerung und Bluetooth-Anzeige der DC-Leistung (Akku) auf dem Handy (iphone/Android). Eine FritzBox angebundene Leistungsüberwachung ist optional möglich (Fritz 210).

Nur weniger 1/3 des erzeugten Stroms könnte ich mit einem "normalen Balkonkraftwerk" ohne Speicher nutzen. Der restliche Strom würde ins Stromnetz "verschenkt." Mit dem Speicher nutze ich viel mehr.
Außerdem kann ich durch den Speicher doppelt so viel Solarpaneele anschließen (vier = 1560Wp).
Also kommt bis zu sechsmal soviel Stromersparnis zusammen.

Als Speicher nutze ich einen Lithium-Ferrophosphat-Akkumulator (LFP-Akku) ergänzt mit einer sehr guten Bluetooth-Überwachung.

Gerne kann man mir eine vorhandene Konfiguration oder eine Planung oder eine Problemstellung beschreiben und ich schauen es mir an und mache gerne passende Vorschläge gemäß meinen Erfahrungen.
https://de.wikipedia.org/wiki/Lithium-Eisenphosphat-Akkumulator
Man steckt das Solarakku-Set einfach zwischen die Leitung von den Solarpaneelen und dem Wechselrichter.
Vom Stromspeicher nutze ich meist 80% 2040 Wh (netto), um deutlich über 5000 Ladezyklen Haltbarkeit zu kommen (das macht die Software in der Steuerungsbox, welche ich konstruiert habe).
Weil nicht jeden Tag im ganzen Jahr ein voller Zyklus genutzt wird,
ist der Speicher auf deutlich über 15 Jahre Nutzung ausgelegt.
Warum sollte sich die Erweiterung mit einem Speicher oder sogar mit mehr Modulen nicht mehrfach wie ein Balkonkraftwerk lohnen? Außerdem macht es viel Spaß, zu sehen, wie wenig Strom man plötzlich vom Netz bezieht.

**Comment by chiquadrat14** [comment]
Danke an alle für das Feedback.

@schlimmchen Vielen Dank, das Teil von Shop Alliance steht jetzt auf meiner Einkaufsliste. Euch weiterhin so viel Engagement wie bisher und hoffentlich bis bald!
